index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>CV of Aleksei Bulgak</title>
+  <link rel="stylesheet" type="text/css" media="screen" href="./css/main.css" />
+</head>
+
+<body>
+  <header>
+    <div class="header container">
```

Comment:
It's better to use common styles like .container first and then use other classes to overwrite needed styles

Reply:
And also it's not necessary to make div with class header if you already have header tag. the same situation with main tag. Also you could use section tag for your blocks